bundles/specmate-logging/src/com/specmate/logging/internal/SpecmateLogReader.java

```diff
@@ -23,8 +30,32 @@
 		level2String.put(LogService.LOG_ERROR, "ERROR");
 	}
 
+	private static Map<String, Integer> string2level = new HashMap<>();
+
+	static {
+		string2level.put("debug", LogService.LOG_DEBUG);
+		string2level.put("info", LogService.LOG_INFO);
+		string2level.put("warning", LogService.LOG_WARNING);
+		string2level.put("error", LogService.LOG_ERROR);
+	}
+
+	/** The log reader service */
 	private LogReaderService logReaderService;
 
+	@Activate
+	public void activate(Map<String, Object> properties) {
+		String confLogLevel = (String) properties.getOrDefault(SpecmateLogReaderConfig.KEY_LOG_LEVEL, "info");

```

Comment:
The name confLogLevel is probably misleading, since it also can be the default value.